internal/provider/resource_aci_endpoint_tag_mac_test.go

```diff
@@ -178,6 +178,10 @@ func TestAccResourceFvEpMacTagWithFvTenant(t *testing.T) {
 					resource.TestCheckResourceAttr("aci_endpoint_tag_mac.test", "id_attribute", "0"),
 					resource.TestCheckResourceAttr("aci_endpoint_tag_mac.test", "name", ""),
 					resource.TestCheckResourceAttr("aci_endpoint_tag_mac.test", "name_alias", ""),
+					resource.TestCheckResourceAttr("aci_endpoint_tag_mac.test", "annotation", "orchestrator:terraform"),
+					resource.TestCheckResourceAttr("aci_endpoint_tag_mac.test", "id_attribute", "0"),
+					resource.TestCheckResourceAttr("aci_endpoint_tag_mac.test", "name", ""),
+					resource.TestCheckResourceAttr("aci_endpoint_tag_mac.test", "name_alias", ""),
```

Comment:
why are we testing this twice?

Reply:
I think they got introduced after rebase. These will anyways disappear once my other PR related to import gets merged